<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CodeTranslator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainVerticalLayout">
    <item>
     <widget class="QTabWidget" name="mainTabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <widget class="QWidget" name="tabTranslate">
       <attribute name="title">
        <string>Translate</string>
       </attribute>
       <layout class="QVBoxLayout" name="tabTranslateLayout">
        <item>
         <layout class="QHBoxLayout" name="comboLayout">
          <item>
           <widget class="QLabel" name="sourceLabel">
            <property name="text">
             <string>Source:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="sourceLangComboBox"/>
          </item>
          <item>
           <widget class="QLabel" name="targetLabel">
            <property name="text">
             <string>Target:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="targetLangComboBox"/>
          </item>
          <item>
           <widget class="QPushButton" name="translateButton">
            <property name="text">
             <string>Translate</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSplitter" name="splitterMain">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QGroupBox" name="inputGroupBox">
           <property name="title">
            <string>Input Code</string>
           </property>
           <layout class="QVBoxLayout" name="inputVBox">
            <item>
             <widget class="QTextEdit" name="inputTextEdit"/>
            </item>
           </layout>
          </widget>
          <widget class="QGroupBox" name="outputGroupBox">
           <property name="title">
            <string>Output Code</string>
           </property>
           <layout class="QVBoxLayout" name="outputVBox">
            <item>
             <widget class="QTextEdit" name="outputTextEdit"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabHistory">
       <attribute name="title">
        <string>History</string>
       </attribute>
       <layout class="QVBoxLayout" name="tabHistoryLayout">
        <item>
         <widget class="QTableWidget" name="historyTableWidget">
          <column>
           <property name="text">
            <string>Source</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Target</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Input</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Output</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Timestamp</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabStats">
       <attribute name="title">
        <string>Stats</string>
       </attribute>
       <layout class="QVBoxLayout" name="tabStatsLayout">
        <item>
         <widget class="QLabel" name="statsLabel">
          <property name="text">
           <string>Translation statistics will appear here.</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignTop|Qt::AlignLeft</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabPlugins">
       <attribute name="title">
        <string>Plugins</string>
       </attribute>
       <layout class="QVBoxLayout" name="tabPluginsLayout">
        <item>
         <widget class="QListWidget" name="pluginListWidget"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
